draft-ietf-taps-impl.md

```diff
@@ -639,20 +639,20 @@ its `Ready` event. This allows the implementation to communicate control data to
 Remote Endpoint that can be used to parse Messages.
 
 ~~~
-MessageFramer.MakeConnectionReady(Connection)
+MessageFramer.MakeConnectionReady(connection)
```

Comment:
Not directly related to parameter names, but perhaps we should add "The framer implementation can indicate that it has finished with this"  **before** the above line. Similar to what we do for the stop event text below.

Reply:
Added!